README.md

```diff
@@ -86,19 +85,20 @@ Run the app locally with
 This will start the local token server and run the app in the development mode. Open [http://localhost:3000](http://localhost:3000) to see the application in the browser.
 
 The page will reload if you make changes to the source code in `src/`.
-You will also see any linting errors in the console. Start the token server locally with
+You will also see any linting errors in the console. If you need to run only the server on its own, you can start the token server locally with
 
     npm run server
```

Comment:
Nit: just noticed that the others have a `$` and these two don't. 
```suggestion
    $ npm run server
```

Same with the one on line 83, which I'm not able to leave a comment on.